docs/mp/jaxrs/12_jaxrs-client.adoc

```diff
@@ -0,0 +1,165 @@
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Jakarta REST (JAX-RS) Client
+:h1Prefix: MP
+:feature-name: Jakarta REST Client
+:description: Jakarta REST (JAX-RS) Client
+:common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
+:rest-client: ../restclient/09_rest-client.adoc
+:jaxrs-spec: https://jakarta.ee/specifications/restful-ws/3.0/jakarta-restful-ws-spec-3.0.html
+:jaxrs-spec-client: https://jakarta.ee/specifications/restful-ws/3.0/jakarta-restful-ws-spec-3.0.html#client_api
+:jaxrs-javadocs: https://jakarta.ee/specifications/restful-ws/3.0/apidocs/
+= :keywords: helidon, rest, jax-rs, client, microprofile, micro-profile
+
+== ToC
+
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<API, API>>
+- <<Configuration, Configuration>>
+- <<Examples, Examples>>
+- <<Additional Information, Additional Information>>
+- <<Reference, Reference>>
+
+== Overview
+
+The Jakarta REST Client defines a programmatic API to access
+REST resources. This API sits at a higher level than traditional HTTP client APIs and
+provides full integration with server-side API concepts like providers. It differs
+from the link:{rest-client}[Rest Client API] in that it does not support
+annotations or proxies, but instead uses builders and a fluent API to
+create and execute requests.
+
+include::{common-deps-page-prefix-inc}[tag=maven-dependency]
+[source,xml]
+----
+     <dependency>
+         <groupId>io.helidon.jersey</groupId>
+         <artifactId>helidon-jersey-client</artifactId>
+     </dependency>
+----
+
+== API
+
+Bootstrapping the API is done by obtaining an instance of `Client`. A single instance of this
+class can be used to create multiple service requests that share the same basic
+configuration, e.g., the same set of _providers_. More precisely, from a `Client` we
+can create multiple `WebTarget` s, and in turn, from each `WebTarget` we can
+create multiple `Invocation` s.
+
+[source,java]
+----
+Client client = ClientBuilder.newClient();
+Response res = client
+    .target("http://localhost:8080/greet")
+    .request("text/plain")
+    .get();
+----
+
+In the snippet above, the call to `target` returns a `WebTarget`, and the call to
+`request` returns an `Invocation.Builder`; finally, the call to `get` returns the `Response`
+that results from accessing the service resource.
+
+Given that this API is fully integrated with message body readers and writers, it is
+possible to request the response body be provided after conversion to a Java type
+-- such as a `String` in the example below.
+
+[source,java]
+----
+Client client = ClientBuilder.newClient();
+String res = client
+    .target("http://localhost:8080/greet")
+    .request("text/plain")
+    .get(String.class);
+----
+
+Alternatively, there are also methods in `Response` that can trigger similar conversions.
+
+Configutation can be specified at the `Client` or `WebTarget` level, both types implement
```

Comment:
"Configutation" -> "Configuration" 

Perhaps add "as" after the comma.

Reply:
Fixed